proto/kvrpcpb.proto

```diff
@@ -791,48 +791,54 @@ message Context {
 // The API version the server and the client is using.
 // See more details in https://github.com/tikv/rfcs/blob/master/text/0069-api-v2.md.
 enum APIVersion {
-    // Mainly for TxnKV and not safe to use RawKV along with TxnKV.
-    //
-    // V1 server only accepts V1 requests. Except that the V1 raw requests with TTL
-    // will be rejected.
+    // `V1` is mainly for TiDB & TxnKV, and is not safe to use RawKV along with the others.
```

Comment:
Is V1 safe to use `TxnKV` along with `TiDB`? In case user may contruct the txn key with same coding method with TiDB. I think user should not use mixed types in V1 mode.

Reply:
It does carry some risk, but we do support it.